### ISSUE TITLE
Missing simple example of how to use form components

### ISSUE THREAD

**Issue by rabe69** [yuche/vue-strap#404]
I try to use a form-group component inside or outside a modal component (current v2 version), but it is not working:
http://jsfiddle.net/rabe69/wxusy5sg/3/

Is there a simple example or a better documentation for this?
I don't understand this part `:valid.sync="valid.user"` 
What's is this `.sync`, I can find all around in this framework?
Where or even must I have to define this `valid`object?

**Comment by wffranco** [comment]
Yep, i know that form-group is not working, but I couldn't fix it yet, don't know what is failing.

**Comment by slava-vishnyakov** [comment]
It's a pity that this doesn't work in Vue2 :(

**Comment by slava-vishnyakov** [comment]
@wffranco I seem to have figured it out:

Here:

https://github.com/yuche/vue-strap/blob/v2/src/Input.vue#L230

the Input component searches for a parent that has `_formValidator` attribute, but FormGroup has only `_formGroup` attribute:

https://github.com/yuche/vue-strap/blob/v2/src/FormGroup.vue#L52

If I do this:

![image](https://cloud.githubusercontent.com/assets/817931/23964089/5d35212c-09c4-11e7-98e4-20c515217d99.png)

(which is probably wrong)

It almose fixes the problem.

Also, I'm not sure what `:valid.sync=` does (doesn't seem to work in Vue2), but this works:

```html
<form-group @isvalid="valid=$event">
```

**Comment by slava-vishnyakov** [comment]
Can't open a pull request, since I'm not sure which is the right fix. For anybody searching for _temporary_ solution - just copy this `FormGroup.vue` from here https://gist.github.com/slava-vishnyakov/530e4b4e10a3b01cc670eded56262c7b to your own project and use it as component.
